### PR TITLE
change: support for building on freebsd amd64

### PR DESCRIPTION
**Pull request by mpraeger**
Fixes issue #2.

### CODE REVIEW

File: tarfs/mknod_stub.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// +build freebsd
```
Comment (vito) [inline tarfs/mknod_stub.go]: Ah, for some reason I thought you were going to make it a no-op. :man_facepalming: In that case let's call this file `mknod_freebsd.go` instead.

**Review by vito — CHANGES_REQUESTED** [review]
tiny tweak requested based on my own misunderstanding, otherwise looks good!

were you able to test this out on freebsd by any chance?

File: tarfs/mknod_stub.go (hunk 1 above)
Comment (mpraeger) [reply tarfs/mknod_stub.go]: No worries, I've renamed the file.

**Review by vito — APPROVED** [review]
:+1: thanks!

**Comment by mpraeger** [comment]
@vito This change results in concourse/fly compiling fine on FreeBSD 11.2-p9 amd64, otherwise the error message linked in issue #2 shows up.

If it actually works I have yet to see; I've not had time to properly setup concourse.

**Pull request merged**